pressure_controller_ros/PressureTrajectoryPoint[] points